cypress/support/commands/lastPayloadsAre.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @jest-environment jsdom
+ */
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Cypress {
+    interface Chainable {
+      lastPayloadsAre: typeof lastPayloadsAre;
+    }
+  }
+}
+
+/**
+ * Make an assertion that the last payload is equal to the expected value.
+ * @param expectedPayloads Expect the last n payloads to be equal to these values.
+ * @returns A chainable cypress command.
+ */
+export const lastPayloadsAre = (expectedPayloads: Array<object>) => {
+  return cy
+    .request(Cypress.env('mockingEndpoint') + '/payloads')
+    .then((response) => {
+      expect(response.body.slice(-expectedPayloads.length)).to.deep.equal(
+        expectedPayloads,
+      );
+    });
+};
```

Comment:
This was the easiest way I could think of having to assert against multiple payloads triggered by a single user action.